applications/CoSimulationApplication/custom_data_structure/pyKratos/Variables.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import print_function, absolute_import, division 
+DISPLACEMENT_X = "DISPLACEMENT_X"
+DISPLACEMENT_Y = "DISPLACEMENT_Y"
+DISPLACEMENT_Z = "DISPLACEMENT_Z"
+DISPLACEMENT = ["DISPLACEMENT",DISPLACEMENT_X, DISPLACEMENT_Y, DISPLACEMENT_Z]
+VELOCITY_X = "VELOCITY_X"
+VELOCITY_Y = "VELOCITY_Y"
+VELOCITY_Z = "VELOCITY_Z"
+VELOCITY = ["VELOCITY",VELOCITY_X,VELOCITY_Y, VELOCITY_Z]
+REACTION_X = "REACTION_X"
+REACTION_Y = "REACTION_Y"
+REACTION_Z = "REACTION_Z"
+REACTION = ["REACTION",REACTION_X, REACTION_Y, REACTION_Z]
+YOUNG_MODULUS = "YOUNG_MODULUS"
+POISSON_RATIO = "POISSON_RATIO"
+DENSITY = "DENSITY"
+VISCOSITY = "VISCOSITY"
+TIME = "TIME"
+DELTA_TIME = "DELTA_TIME"
+TEMPERATURE = "TEMPERATURE"
+CONDUCTIVITY = "CONDUCTIVITY"
+VOLUME_HEAT_SOURCE = "VOLUME_HEAT_SOURCE"
+PRESSURE = "PRESSURE"
+BODY_FORCE_X ="BODY_FORCE_X"
+BODY_FORCE_Y = "BODY_FORCE_Y"
+BDF_COEFFICIENTS = "BDF_COEFFICIENTS"
+IS_LAGRANGIAN = "IS_LAGRANGIAN"
+EXTERNAL_FORCE_X = "EXTERNAL_FORCE_X"
+EXTERNAL_FORCE_Y = "EXTERNAL_FORCE_Y"
+EXTERNAL_FORCE_Z = "EXTERNAL_FORCE_Z"
+EXTERNAL_FORCE = ["EXTERNAL_FORCE",EXTERNAL_FORCE_X, EXTERNAL_FORCE_Y, EXTERNAL_FORCE_Z]
+EXTERNAL_UNIT_DENSITY = "EXTERNAL_UNIT_DENSITY"
+FACE_HEAT_FLUX = "FACE_HEAT_FLUX"
+FSI_INTERFACE = "FSI_INTERFACE"
```

Comment:
I guess we could generate this file automatically using KRatos variables (i am speaking of generating it "offline")